app/Entities/Repos/PageRepo.php

```diff
@@ -148,6 +148,12 @@ public function getNewDraftPage(Entity $parent)
             $page->book_id = $parent->id;
         }
 
+        if ($page->book->defaultTemplate) {
+            $page->forceFill([
+                'html'  => $page->book->defaultTemplate->html,
+            ]);
+        }
+
```

Comment:
This currently ignores permissions. I think we'll need to respect page permissions, so view access is required for the template to be used, otherwise other side-effects will be introduced (Image access permissions for example).